lib/neurow/public_api.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Neurow.PublicApi do
+  require Logger
+  import Plug.Conn
+  use Plug.Router
+
+  plug(:monitor_sse)
+
+  plug(Neurow.JwtAuthPlug,
+    allowed_algorithm: "RS256",
+    jwk_provider: &Neurow.Configuration.public_issuer_jwks/1,
+    max_lifetime: 60 * 2,
+    audience: "public_api"
+  )
+
+  plug(:match)
+  plug(:dispatch)
+
+  get "v1/subscribe" do
+    case conn.assigns[:jwt_payload] do
+      %{"iss" => issuer, "sub" => sub} ->
```

Comment:
The idea here is to use the issuer as a way to isolate multiple systems that use neurow for each other.